src/tests/test08-test-configs.sh

```diff
@@ -98,23 +104,16 @@ daemon = "on"
 write-ip = off
 write-proxy = on
 EOF
-hitch --test --config=$CONFFILE ${CERTSDIR}/default.example.com
-test $? -eq 0 || die "Valid config test08d unparseable?"
 
 # Issue #52.
-hitch --config=${CONFDIR}/default.cfg --help
-test $? -eq 0 || die "--help after --config does not work as expected."
+run_cmd hitch --config=${CONFDIR}/default.cfg --help
 
+# XXX: unclear check
```

Comment:
See the referenced issue #52 (and #22).

You seem to have removed the successfull return check after the `--config=... --help` invocation, I suggest adding that back in and also leaving the `--test --config=...` test in place.

Reply:
I'm confused by the error message not matching the actual command (`--test` vs `--help`) but with #52 and #22 I can see it was probably a matter of copy/pasta. I will re-enable this check, thanks.